### PR TITLE
update code owners based on team changes

### PR DESCRIPTION
**Pull request by davidkline-ms**
Remove exited team members, add new owners

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -18,62 +18,53 @@
 # first in this file, followed by more specific rules.
 
 # Build / CI Pipeline YAML
-/pipelines/ @wiwei @MenelvagorMilsom @macborow
+/pipelines/ @MenelvagorMilsom @macborow
 
 # Build / CI C#
-/Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/ @wiwei @MenelvagorMilsom @macborow
+/Assets/MRTK/codeowners/Utilities/BuildAndDeploy/ @MenelvagorMilsom @macborow
```
Comment (keveleigh) [inline CODEOWNERS:24]:
```suggestion
/Assets/MRTK/Core/Utilities/BuildAndDeploy/ @MenelvagorMilsom @macborow
```

File: CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# This file contains information about the code owners for various
+H# This file contains information about the code owners for various
```
Comment (keveleigh) [inline CODEOWNERS:1]:
```suggestion
# This file contains information about the code owners for various
```

File: CODEOWNERS [hunk 3]
```diff
@@ -18,62 +18,53 @@
...
 
 # Build / CI Powershell Scripts
-/scripts/ci/ @wiwei @MenelvagorMilsom @macborow
+/scripts/ci/ @MenelvagorMilsom @macborow
 
 # Packaging files
-*.asmdef @wiwei @davidkline-ms @keveleigh @andreiborodin
-*.nupkg @wiwei @andreiborodin @davidkline-ms
+*.asmdef @davidkline-ms @keveleigh
+*.nupkg @davidkline-ms
```
Comment (keveleigh) [inline CODEOWNERS:31]: Add a new line for the UPM templates?

Comment (davidkline-ms) [reply CODEOWNERS:31]: good thought!

**Review by CDiaz-MS — APPROVED** [review]

**Pull request merged**